samples/peripheral/radio_test/src/radio_cmd.c

```diff
@@ -1274,6 +1278,34 @@ SHELL_STATIC_SUBCMD_SET_CREATE(sub_fem,
 );
 #endif /* CONFIG_FEM */
 
+#if CONFIG_RADIO_TEST_POWER_CONTROL_AUTOMATIC
+static int cmd_total_output_power_set(const struct shell *shell, size_t argc, char **argv)
+{
+	int power;
+
+	if (argc == 1) {
+		shell_help(shell);
+		return SHELL_CMD_HELP_PRINTED;
+	}
+
+	if (argc > 2) {
+		shell_error(shell, "%s: bad parameters count", argv[0]);
+		return -EINVAL;
+	}
+
+	power = atoi(argv[1]);
+
+	if ((power > INT8_MAX) || (power < INT8_MIN)) {
+		shell_error(shell, "%s: Out of range power value", argv[0]);
+		return -EINVAL;
+	}
+
+	config.txpower = (uint8_t)power;
```

Comment:
It really doesn't matter here, the `config.txpower` had unsigned type probably from legacy reason but it can be now a signed type for better readability. I changed it in all places to `int8_t`.